Source/CarthageKit/Project.swift

```diff
@@ -458,6 +459,21 @@ public final class Project {
 				return copyProduct(dSYMURL, resolvedDestinationURL)
 			}
 	}
+	
+	/// Copies any *.bcsymbolmap files matching the given framework and contained
+	/// within the given directory URL to the directory that the framework
+	/// resides within.
+	///
+	/// If no bcsymbolmap files are found for the given framework, completes with
+	/// no values.
+	///
+	/// Sends the URLs of the bcsymbolmap files after copying.
+	public func copyBCSymbolMapsToBuildFolderForFramework(frameworkURL: NSURL, fromDirectoryURL directoryURL: NSURL) -> SignalProducer<NSURL, CarthageError> {
+		return BCSymbolMapsForFramework(frameworkURL)
```

Comment:
What archives are created that way? And I'm not sure how Carthage can support that anyway. If it doesn't know where to find the bcsymbolmaps, then it can't possibly install them, and the behavior will be the same as if the archive doesn't have bcsymbolmaps included at all.